packages/@okta/vuepress-site/docs/guides/submit-app/main/saml2/submit.md

```diff
@@ -1,73 +1,76 @@
 
-* **What type(s) of sign-in flows do you support?** &mdash; choose from IdP or SP initiated, or both. If you support SP-initiated flows, you must specify the URL used to initiate the SP flow and include a description of how the SP flow is triggered by your integration.
+* **What type(s) of sign-in flows do you support?**: Choose from IdP or SP initiated, or both. If you support SP-initiated flows, you must specify the URL used to initiate the SP flow and include a description of how the SP flow is triggered by your integration.
 
-* **Force authentication?** &mdash; select **Yes** if your application forces users to authenticate again, even if they have an existing session. If you select **Yes**, include how customers configure force authentication on your end? Provide the steps for Okta to test this forced authentication.
+* **Force authentication?**: Select **Yes** if your application forces users to authenticate again, even if they have an existing session. If you select **Yes**, include how customers configure force authentication on your end? Provide the steps for Okta to test this forced authentication.
 
-* **Do you use encrypted assertions?** &mdash; select **Yes** if you support an encryption method for authentication assertion.
+* **Do you use encrypted assertions?**: Select **Yes** if you support an encryption method for authentication assertion.
 
-  * **Encryption Algorithm** &mdash; select which standard algorithm is used in your encryption assertion.
-  * **Key Transport Algorithm** &mdash; select the key transport algorithm used in your encryption.
-  * **Are you going to use the same encryption certificate for all tenants?** &mdash; if you support the same encryption certificate for all tenants, copy your certificate and paste it in the field provided.
+  * **Encryption Algorithm**: Select which standard algorithm is used in your encryption assertion.
+  * **Key Transport Algorithm**: Select the key transport algorithm used in your encryption.
+  * **Are you going to use the same encryption certificate for all tenants?**: If you support the same encryption certificate for all tenants, copy your certificate and paste it in the field provided.
 
-* **Do you support SP-initiated Single-Logout?** &mdash; if you select **Yes**, then **Are you going to use the same single-logout certificate for all tenants?**
+* **Do you support SP-initiated Single-Logout?**: If you select **Yes**, then **Are you going to use the same single-logout certificate for all tenants?**
 
     If you support the same single-logout certificate for all tenants, copy your certificate and paste it in the field provided.
 
     The Single Logout URL and SP Issuer should be specified in the test application.
 
-    >**Note:** Okta only supports signed logout requests.
+    >**Note:** Okta only supports single logout requests.
```

Comment:
@ScottRussell-okta : I updated "signed" to "single" in this note - please confirm if this correction. Thanks

Reply:
Good catch, thank you.  Yes that is correct.